README.rst

```diff
@@ -2,14 +2,6 @@
 Overview
 ########
 
-**WARNING: As previously announced on our communication channels, the Kytos
-project will enter the "shutdown" phase on May 31, 2021. After this date,
-only critical patches (security and core bug fixes) will be accepted, and the
-project will be in "critical-only" mode for another six months (until November
-30, 2021). For more information visit the FAQ at <https://kytos.io/faq>. We'll
-have eternal gratitude to the entire community of developers and users that made
-the project so far.**
-
 Attention!
 
 THIS IS AN EXPERIMENTAL NAPP AND ITS EVENTS, METHODS AND STRUCTURES MAY CHANGE A LOT ON THE NEXT FEW DAYS/WEEKS, USE IT AT YOUR OWN DISCERNMENT
```

Comment:
@ajoaoff are we going to also set this NApp as stable? I just realized the experimental paragraph in the README

Reply:
I don't think we are testing this, not sure if we can consider it to be stable.

Reply:
right @ajoaoff